package.json

```diff
@@ -78,9 +64,28 @@
     "less-plugin-clean-css": "^1.5.1",
     "mini-css-extract-plugin": "^0.9.0",
     "prettier": "^1.19.1",
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0",
```

Comment:
These appear in both devDeps and peerDeps.  The former is avoid missing peerDeps when installing things like enzyme.  This is necessary because the installation of devDeps and peerDeps are separate steps and devDeps installation happens first.  This isn't necessary, since the missing peerDeps warnings are just warnings, but it is nice to do `yarn install` and not get a wall of warnings.